objects/src/errors.rs

```diff
@@ -95,15 +96,13 @@ impl fmt::Display for AccountDeltaError {
 pub enum AssetError {
     AmountTooBig(u64),
     AssetAmountNotSufficient(u64, u64),
-    FungibleAssetInvalidFirstBit,
```

Comment:
Remove these errors since:

- The first bit is now used to represent the storage
- This was a weird coupling of the account id format and the error type